src/ReactiveDomain.nuspec

```diff
@@ -9,11 +9,11 @@
     <description>Package includes all ReactiveDomain Core assemblies</description>
     <copyright>Copyright © 2014-2022 PerkinElmer Inc., Linedata Inc.</copyright>
     <dependencies>
-      <group targetFramework=".NETStandard2.0">
-        <dependency id="EventStore.Client" version="5.0.1" exclude="Build,Analyzers" />
+      <group targetFramework="netstandard2.0">
+        <dependency id="EventStore.Client" version="21.2.2" exclude="Build,Analyzers" />
         <dependency id="EventStore.Client.Embedded" version="5.0.1" exclude="Build,Analyzers" />
```

Comment:
Should this be removed or updated?

Reply:
neither :-(
it is not published in later versions, and it used in the unit test framework to confirm the functionality of the mock store tests. 
Some version of this will be re-added in a future version, we can update it then.